builds/v2.0/entrypoint.mjs

```
@@ -0,0 +1,593 @@
+      opt.value = name;
+      opt.textContent = name;
+      opt.selected = name === current;
+      select.appendChild(opt);
+    }
+    scroll.appendChild(makeRow("Build Channel", select));
+
+    const info = document.createElement("p");
+    info.style.cssText = "margin:2px 8px 10px;font-size:0.72rem;color:rgba(255,255,255,0.35);line-height:1.5;";
+    const updateInfo = () => {
+      const hosts = map[select.value];
+      if (!hosts) return (info.textContent = "");
```

Biome (2.4.4)

[error] The assignment should not be in an expression.

(lint/suspicious/noAssignInExpressions)